daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2471,6 +2472,7 @@ func (devices *DeviceSet) Status() *Status {
 		}
 	}
 
+	status.MinFreeSpace = fmt.Sprintf("%v%%", devices.minFreeSpacePercent)
```

Comment:
Just a thought; Perhaps we should show both the absolute size (xx Gb) and percentage, something like `123 Gb (12%)`